osu.Game/Storyboards/Drawables/DrawableStoryboardSprite.cs

```diff
@@ -120,8 +120,13 @@ private void load(TextureStore textureStore, Storyboard storyboard)
             var drawable = storyboard.CreateSpriteFromResourcePath(Sprite.Path, textureStore);
 
             if (drawable != null)
+            {
                 InternalChild = drawable;
 
+                if (Sprite.Path == storyboard.BeatmapInfo.Metadata?.BackgroundFile)
+                    drawable.ScaleTo(480f / drawable.Height);
```

Comment:
Can this be specified in `DimmableStoryboard` rather than here? I would rather avoid relying on matching paths for sizing like this.